### PR TITLE
Fix jupyterlite_contents

### PR DESCRIPTION
**Pull request by jasongrout**
It seems we are only getting one file in the jlite contents these days, which started happening right around the time we switched to jupyterlite_sphinx 0.7.0.

### CODE REVIEW

**Review by martinRenou — APPROVED** [review]
Nice catch! Thanks

**Comment by jasongrout** [comment]
Docs is failing to build:

```
yarn install v1.21.1
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
error An unexpected error occurred: "https://registry.npmjs.org/corser/-/corser-2.0.1.tgz: Request failed \"503 Service Unavailable\"".
```
So I'll try again in a bit.

File: docs/source/conf.py [hunk 1]
```diff
@@ -31,7 +31,7 @@
 # "examples/Layout Example.ipynb", # bqplot/ipyleaflet dependencies require ipywidgets 7
 # "examples/Layout Templates.ipynb", # bqplot/ipyleaflet dependencies require ipywidgets 7
 # "examples/Variable Inspector.ipynb", # uses ipykernel-specific features
-jupyterlite_contents = "examples"
+jupyterlite_contents = "examples/"
```
Comment (jasongrout-db) [inline docs/source/conf.py:34]:
```suggestion
jupyterlite_contents = "examples"
```

**Comment by jasongrout** [comment]
Looks like the actual error was that jupyterlite_contents inadvertently can't be a string anymore, so I changed it to an array. I put in a fix for jupyterlite_sphinx in https://github.com/jupyterlite/jupyterlite-sphinx/pull/73

**Comment by jasongrout** [comment]
> I put in a fix for jupyterlite_sphinx in [jupyterlite/jupyterlite-sphinx#73](https://github.com/jupyterlite/jupyterlite-sphinx/pull/73)

And @martinRenou is amazingly fast at releasing, and the fix is now in jupyterlite-sphinx 0.7.2. Thanks Martin!

File: docs/source/jupyter_lite_config.json [hunk 2]
```diff
@@ -68,7 +68,7 @@
       "https://files.pythonhosted.org/packages/py3/t/traitlets/traitlets-5.1.1-py3-none-any.whl",
       "https://files.pythonhosted.org/packages/py3/t/typing_extensions/typing_extensions-4.1.1-py3-none-any.whl",
       "https://files.pythonhosted.org/packages/py3/v/vega-datasets/vega_datasets-0.9.0-py3-none-any.whl",
-      "https://files.pythonhosted.org/packages/py3/w/widgetsnbextension/widgetsnbextension-4.0.0rc1-py3-none-any.whl",
+      "https://files.pythonhosted.org/packages/py3/w/widgetsnbextension/widgetsnbextension-4.0.0rc2-py3-none-any.whl",
```
Comment (vidartf) [inline docs/source/jupyter_lite_config.json:71]: Cannot this also be done with a `../../python/...` path?

**Review by vidartf — COMMENTED** [review]
Mostly looks good. Can we use a local path for the nbextension? If not, we will need to update that line for every release, right?

File: docs/source/jupyter_lite_config.json (hunk 2 above)
Comment (jasongrout) [reply docs/source/jupyter_lite_config.json]: Great question. I've pushed a commit to try it.

Comment (jasongrout) [reply docs/source/jupyter_lite_config.json]: Seems to work great!

**Review by vidartf — APPROVED** [review]

**Pull request merged**